src/main/java/com/techsophy/tsf/account/service/impl/ACLServiceImpl.java

```diff
@@ -99,18 +101,32 @@ public ACLSchema getACLById(String id)
     public ACLValidate checkACLAccess(String id, CheckACLSchema checkACLSchema) throws JsonProcessingException, AccessDeniedException
     {
         Map<String,Object> userDetailsFromKeycloak= tokenUtils.getUserInformationMap(tokenUtils.getTokenFromContext());
+        AuditableData auditableData = userFormDataService.getUserFormDataByUserId(userDetailsFromKeycloak.get("userId")+"",false);
+        UserFormDataSchema userFormDataSchema = objectMapper.convertValue(auditableData,UserFormDataSchema.class);
+        RequestProperties requestProperties = new RequestProperties();
+        requestProperties.setUserInfo(userDetailsFromKeycloak);
+        requestProperties.setContext(checkACLSchema.getContext()+"");
+        if(userFormDataSchema!=null&&userFormDataSchema.getUserData()!=null) {
+        Map<String,Object> userData = userFormDataSchema.getUserData();
+        requestProperties.setUserId(userFormDataSchema.getUserId());
```

Comment:
see if you can use https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/index.html?org/apache/commons/lang3/StringUtils.html